.script/utils/gitWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import { cli, devOps } from '@azure/avocado';
+import * as logger from './logger';
+import './stringExtenssions'
+
+export async function GetDiffFiles(fileTypeSuffixes?: string[], filePathFolderPreffixes?: string[]){
+    const config = cli.defaultConfig();
+    const pr = await devOps.createPullRequestProperties(config);
+  
+    if (pr === undefined) {
+      console.log("undifined PR");
```

Comment:
typo - undefined

minor - maybe a more informative error, like - "Could not create a PR, exiting. If issue persists - please open an issue"

Reply:
Fix.
according to PR function documentetion "The function returns undefined if it's not Azure DevOps CI for a Pull Request."